core/domain/exp_domain.py

```diff
@@ -1647,11 +1701,13 @@ def create_default_state(
         """
         content_html = (
             feconf.DEFAULT_INIT_STATE_CONTENT_STR if is_initial_state else '')
+        content_id = feconf.DEFAULT_NEW_STATE_CONTENT_ID
```

Comment:
Is this really a default, or is it just the permanent content_id for all states? Because this won't ever change, right? So maybe just call it STATE_CONTENT_ID or similar.

Reply:
I'll do this after finalizing the structure of NEW_STATE_TEMPLATE.